### DISCUSSION TITLE
谷歌账号无法登录ChatGPT

### DISCUSSION THREAD

**Discussion by Sunsh4j** [tw93/Pake#565] [Q&A]
如题，在安装ChatGPT后，无法登录谷歌账号，从而无法登录绑定的ChatGPT账号
![image](https://github.com/tw93/Pake/assets/43169575/a054ebb5-d0d6-4906-b7e9-3b58903cebf9)

**Comment by Samiya321** [comment]
+1 同无法登录

**Reply by tw93** [reply]
请问是Mac系统吗？我试了试我的Mac系统是ok的

**Reply by Sunsh4j** [reply]
win10 ltsc

**Reply by Sunsh4j** [reply]
之前安装后登录成功过，前几天再次安装后，不知道为啥无法登录了

**Reply by tw93** [reply]
大概知道了 试试换一个网络vpn地址试试

**Reply by Sunsh4j** [reply]
还是不行，我在浏览器上可以正常登录啊

**Reply by Samiya321** [reply]
发现是IPV6的问题

**Reply by Sunsh4j** [reply]
怎么说？要开ipv6吗？
